script.js

```diff
@@ -24,8 +22,7 @@ taskList.addEventListener("click", function (event) {
         const checkbox = event.target;
         const taskText = checkbox.nextElementSibling;
         const listItem = checkbox.parentNode;
-        const changeTo= checkbox.checked;
-        editDb(listItem.id, changeTo);
+        const changeTo = checkbox.checked;
         if (checkbox.checked) {
```

Comment:
add comments for this part of the code to expain what you are doing

Reply:
OKII